charts/ingress-nginx/README.md

```diff
@@ -219,3 +219,46 @@ Error: UPGRADE FAILED: Service "?????-controller" is invalid: spec.clusterIP: In
 Detail of how and why are in [this issue](https://github.com/helm/charts/pull/13646) but to resolve this you can set `xxxx.service.omitClusterIP` to `true` where `xxxx` is the service referenced in the error.
 
 As of version `1.26.0` of this chart, by simply not providing any clusterIP value, `invalid: spec.clusterIP: Invalid value: "": field is immutable` will no longer occur since `clusterIP: ""` will not be rendered.
+
+## Using custom default backend
```

Comment:
Please don't add this information here. If you want to add another example, please open a different PR updating https://kubernetes.github.io/ingress-nginx/user-guide/default-backend/

Reply:
Fixed, I think I will drop that for now.